Public/Src/Engine/Cache/Artifacts/LocalDiskContentStore.cs

```diff
@@ -207,7 +207,6 @@ public sealed class LocalDiskContentStore : ILocalDiskFileSystemView
                         : CreateSymlinkIfNotExistsOrTargetMismatch(expandedPath.Path, reparsePointInfo.Value.GetReparsePointTarget());
                 }
 
-                // TODO:58494: We shouldn't need to open a new handle here. In fact, that is a race, and why there isn't a FileContentTable.RecordContentHashAsync taking a path.

```

Comment:
We still open a new handle here, right?  So this comment still seems relevant

Reply:
The comment is present inside the actual call inside `TryOpenAndTrackPathAsync`